perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -298,13 +310,13 @@ def Prepare(benchmark_spec):
       fill_size = FLAGS.device_fill_size
     else:
       fill_path = posixpath.join(mount_point, DEFAULT_TEMP_FILE_NAME)
-      if FLAGS['disk_fill_size'].present:
-        fill_size = FLAGS.disk_fill_size
+      if FLAGS['device_fill_size'].present:
```

Comment:
Yes, it is. There's another one in this PR too. I can move them to a separate PR if you want.